caiman/source_extraction/volpy/volpy_gui.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug  6 10:48:16 2020
+VolPy GUI interface for neuron selection
+@author: @caichangjia
+"""
+import cv2
+import numpy as np
+import os
+import pyqtgraph as pg
+from pyqtgraph import FileDialog
+from pyqtgraph.Qt import QtGui
+from pyqtgraph.parametertree import Parameter, ParameterTree
+import random
+from skimage.draw import polygon
+
+import caiman as cm
+from caiman.external.cell_magic_wand import cell_magic_wand_single_point
+
+try:
+    cv2.setNumThreads(1)
+except:
+    print('Open CV is naturally single threaded')
+
+try:
+    if __IPYTHON__:
+        print(1)
+        # this is used for debugging purposes only. allows to reload classes
```

Comment:
Instead of "allows to reload classes":
"Automatically reloads imports if they change"